src/main/git-elegant-feature

```diff
@@ -3,6 +3,14 @@ set -e
 
 default() {
     _validate "$1" "Feature name"
+    local status="git status | grep 'Changes to be commited: || Changes not staged for commit:'"
+    if [ -n $(eval "$status") ]; then
+        git stash
+    fi
+
     git-elegant pull $MASTER
     git checkout -b "$1"
+    if [[ $(git stash list) ]]; then
```

Comment:
@ivan-podorozhnyi On this check might be some mess, if no stash was added but list contain some another stashes.